test/net/sourceforge/kolmafia/request/AscensionRequestTest.java

```diff
@@ -0,0 +1,66 @@
+package net.sourceforge.kolmafia.request;
+
+import static internal.helpers.Networking.html;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import net.sourceforge.kolmafia.KoLCharacter;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class AscensionRequestTest {
+  @BeforeEach
+  public void initializeCharPrefs() {
+    KoLCharacter.reset("fakePrefUser");
+    KoLCharacter.reset(true);
+    Preferences.saveSettingsToFile = false;
+  }
+
+  @AfterEach
+  public void resetCharAndPrefs() {
+    KoLCharacter.reset("");
+    KoLCharacter.reset(true);
+    KoLCharacter.setUserId(0);
+    Preferences.saveSettingsToFile = false;
+  }
+
+  @Test
+  void testOnAscension() {
```

Comment:
What is this? Is it possible to split all these requests into different tests? If not why not?

Reply:
Tests has been seperated